### PR TITLE
Contao 5

### PR DESCRIPTION
**Pull request by contaoacademy**
Add Changes from https://github.com/dklemmt/contao_dk_mmenu/compare/master...saxony-switzerland-developers:contao_dk_mmenu:master

### CODE REVIEW

File: src/Helper/MmenuHelper.php [hunk 1]
```diff
@@ -119,6 +119,7 @@ public static function processModuleSettings(Module $module, string $jsTemplateN
         $jsTemplate->configuration = $configuration;
 
         // Add module JavaScript (and replace insert tags, see #66)
-        $GLOBALS['TL_BODY'][] = Controller::replaceInsertTags($jsTemplate->parse());
+        //Revert to the original without inserttags
+        $GLOBALS['TL_BODY'][] = $jsTemplate->parse();
```
Comment (rabauss) [inline src/Helper/MmenuHelper.php:123]: This was a user requirement from #66 - https://github.com/dklemmt/contao_dk_mmenu/pull/66#issuecomment-1345256801

Comment (contaoacademy) [reply src/Helper/MmenuHelper.php:123]: Yes I know, but at the moment I have no idea how to implement the  newinserttagparser service. That would have to be done by someone with more programming experience ;-)

**Comment by fritzmg** [comment]
Superseded by 6cc8509fc718293cb208e2e933ca7366aedfad75

**Comment by contaoacademy** [comment]
Danke

**Pull request closed without merging**